core/src/main/java/jenkins/model/ParameterizedJobMixIn.java

```diff
@@ -303,16 +296,42 @@ public final String getBuildNowText() {
     }
 
     /**
-     * Marker for job using this mixin.
+     * Marker for job using this mixin, and default implementations of many methods.
      */
-    public interface ParameterizedJob extends hudson.model.Queue.Task, hudson.model.Item {
+    public interface ParameterizedJob<JobT extends Job<JobT, RunT> & ParameterizedJobMixIn.ParameterizedJob<JobT, RunT> & Queue.Task, RunT extends Run<JobT, RunT> & Queue.Executable> extends BuildableItem {
```

Comment:
Well the mixin Javadoc does say

> be scheduled in various ways

which I guess is enough. Anyway `BuildableItem` is an interface which is not used all that much and not especially interesting to people grokking this code.